### DISCUSSION TITLE
power vs dc power

### DISCUSSION THREAD

**Discussion by muibusan** [tbnobody/OpenDTU#558] [Q&A] [answered]
Hi!

Im wondering why is **power** different from **power dc**? Please compare the attached screenshot.

![OpenDTU Live Data](https://user-images.githubusercontent.com/16862260/216815494-0db1117c-6dad-4087-911f-ed062093fdd2.png)

**Comment by softwm8818** [comment]
ever heard of efficiency...

**Comment by ghost** [comment, marked as answer]
"DC Power" is the panel-delivered inverter input power. "Power" means what the inverter outputs to the grid. Both values are different since no process is free of losts. The term "efficiency" describes the relationship between both.

**Reply by muibusan** [reply]
Thank you very much. I wasn't aware the inverter delivers both of that data. So, in the end I need "power" for any analysis.
